doc/benchmarking/benchmarking_tutorial.rst

```diff
@@ -3,7 +3,7 @@ Benchmarking
 
 .. note:: This is the new benchmarking method only available in ROS Kinetic, onward.
 
-.. note:: To use this benchmarking method, you will need to download and install the ROS Warehouse plugin. Currently this is not available from Debians and requires a source install for at least some aspects. For source instructions, see `this page <http://moveit.ros.org/install/source/dependencies/>`_
+.. note:: The default method for initializing the planning scene uses the ROS Warehouse plugin. Currently this is not available from Debians and requires a source install for at least some aspects. For source instructions, see `this page <http://moveit.ros.org/install/source/dependencies/>`_
```

Comment:
In Melodic, there are released packages `moveit_ros_warehouse` and `warehouse_ros`. So, is this comment still valid?

Reply:
You're right, it's `moveit_ros_warehouse` now.

Reply:
Ok, I removed this note and also the note above (this benchmark method has been new for a while now)